phoenix-core/src/test/java/org/apache/phoenix/util/TestUtil.java

```diff
@@ -877,8 +875,12 @@ public static void dumpTable(Table table) throws IOException {
     }
 
     public static int getRawRowCount(Table table) throws IOException {
+        return getRawRowCount(table, true);
+    }
+
+    public static int getRawRowCount(Table table, boolean isRaw) throws IOException {
```

Comment:
nit: Seems a little odd to have a function named getRawRowCount that doesn't do a raw scan if you pass in false. (As opposed to having a getRowCount funciton with an isRaw parameter.)